operations/rates.md

```diff
@@ -104,6 +104,116 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](accountingcategories.md#accounting-category) the rate belongs to. |
+| `BusinessSegmentId` | string | optional | Unique identifier of the [Business segment](businesssegments.md#business-segment). |
+| `Names` | [Localized text](_objects.md#localized-text) | required | All translations of the name of the rate. |
+| `ShortNames` | [Localized text](_objects.md#localized-text) | optional | All translations short name of the rate. |
+| `ExternalNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the external name of the rate. |
+| `Descriptions` | [Localized text](_objects.md#localized-text) | optional | All translations of the description. |
+| `PricingType` | [Pricing type](rates.md#pricing-type) | required | Discriminator in which field inside `Pricing` contains additional data. |
+| `Pricing` | [Pricing](rates.md#rate-pricing-parameters) | required | Contains additional data about pricing of the rate. |
+
+### Rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRatePricing` | [Base rate pricing](rates.md#base-rate-pricing-parameters) | optional | Additional data for rate with base rate pricing. |
+| `DependentRatePricing` | [Base rate pricing](rates.md#dependent-rate-pricing-parameters) | optional | Additional data for rate with dependent rate pricing. |
+
+### Base rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Amount` | [Amount parameters](orders.md#amount-parameters) | required | Unit amount of the product. Note this overrides any previously defined amount. |
+| `NegativeOccupancyAdjustment` | decimal | required | This is the amount added to the price when occupancy of the space is less than the Space Category _Capacity_. To provide a discount price for under-occupancy, simply use a negative value. |
+| `ExtraOccupancyAdjustment` | decimal | required | This is the amount added to the price when the Space Category _Capacity_ is exceeded. |
+
+### Dependent rate pricing parameters
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BaseRateId` | string | required | Unique identifier of the rate on which this dependent rate is based. |
+| `RelativeAdjustment` | decimal | required | This value is multiplied by the base rate to calculate an adjustment added to the base rate, e.g. a value of `1.0` means add 100% of the base rate to the base rate, effectively doubling the price. To subtract an amount, simply use a negative value, e.g. a value of `-0.1` means subtract 10% of the base rate from the base rate. |
+| `AbsoluteAdjustment` | decimal | required | This value is simply added to the base rate to obtain the new price. To subtract an amount, use a negative value. |
+
+### Response
+
+```javascript
+{
+    "Rates": [
+        {
+            "BaseRateId": null,
```

Comment:
Sorry to be annoying, but can we put the JSON properties in the same order as they are in the table definition, e.g. starting with `Id`?